modules/virt-about-node-labeling-cpu-features.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+// * virt/node_maintenance/virt-managing-node-labeling-obsolete-cpu-models.adoc
+
+[id="virt-about-node-labeling-cpu-features_{context}"]
+= About node labeling for CPU features
+
+Through the process of iteration, the base CPU features in the minimum CPU model are eliminated from the list of labels generated for the node.
+
+For example, an environment might have two supported CPU models: `Penryn` and `Haswell`. If `Penryn` is specified as the CPU model for `minCPU`, each base CPU feature for `Penryn` is compared to the list of CPU features supported by `Haswell`.
+
+If both `Penryn` and `Haswell` support a specific CPU feature, a label is not created for that feature. Labels are generated for CPU features that are supported only by `Haswell` and not by `Penryn`. For example:
```

Comment:
I think the logic gets muddied by the examples. Pity we can't use a venn diagram. 

Maybe bullet points would help the user to understand the relationship between the CPUs and their features.

- Environment supports P and H.
- P is the minimum CPU model, so its features are eliminated from the label list.
- P and H both support feature X.
- H supports only feature Y.
- Therefore, a label is generated for feature Y, but not for feature X.

Reply:
I'll add some bullet points. You're right that a venn diagram would be more straightforward :)